### ISSUE TITLE
broken kube api link in API reference

### ISSUE THREAD

**Issue by judovana** [konflux-ci/architecture#207]
in  https://konflux-ci.dev/architecture/ref/application-environment-api.html#snapshotlist , there is (one of firsts) paragraph of 
```
Application

Application is the Schema for the applications API. For a detailed description with examples, refer to Hybrid Application Service Kube API 
```

Where `Hybrid Application Service Kube API  ` is link to https://github.com/konflux-ci/architecture/blob/main/book/HAS/hybrid-application-service-api.md  which is unluckily broken.

**Comment by judovana** [comment]
Any updates here?

**Comment by arewm** [comment]
@dirgim @davidmogar @mmorhun , do you know where the relevant API can be found now? Has it been moved since we deprecated HAS within Konflux?

**Comment by arewm** [comment]
@judovana , do you know if the link to https://github.com/konflux-ci/architecture/blob/main/book/HAS/hybrid-application-service-api.md is still present? I couldn't find it off hand. I think that it will need to be replaced with 
https://github.com/konflux-ci/architecture/blob/main/architecture/hybrid-application-service.md

**Comment by judovana** [comment]
yes the (broken )link  this issue is about is still there.  If the one you suggest as replacement is correct one, IDK:(

**Comment by ralphbean** [comment]
Just today in the arch call, @adambkaplan proposed an update to the way we generate api docs which will be relevant here.

**Comment by adambkaplan** [comment]
Pull request with the website API docs generator is here: https://github.com/konflux-ci/docs/pull/236

**Comment by ralphbean** [comment]
I posted https://github.com/konflux-ci/application-api/pull/87.
